committee-steering/governance/sig-governance.md

```diff
@@ -117,13 +116,21 @@ curation from other SIG participants
 
 ### Tech Lead
 
-- *Optional Role*: SIG Technical Leads
-  - Establish new subprojects
-  - Decommission existing subprojects
-  - Resolve X-Subproject technical issues and decisions
-  - Number: 2-3
-  - Membership tracked in [sigs.yaml]
-  - Role description in [technical-lead.md]
+- Number: 2+
```

Comment:
is it not optional any longer? nice

Reply:
Correct - with it no longer rolling up to chair, it needs to be explicitly defined (even if folks are serving both roles). We'll realistically have a good chunk of people acting as both chair and tl for awhile, but hopefully more folks will split responsibilities.